src/mono/wasm/runtime/src/mono/debug.ts

```diff
@@ -51,8 +51,8 @@ function mono_wasm_malloc_and_set_debug_buffer(command_parameters: string) {
             Module._free(_debugger_buffer);
         _debugger_buffer_len = Math.max(command_parameters.length, _debugger_buffer_len, 256);
         _debugger_buffer = Module._malloc(_debugger_buffer_len);
-        _debugger_heap_bytes = new Uint8Array(Module.HEAPU8.buffer, _debugger_buffer, _debugger_buffer_len);
     }
+    _debugger_heap_bytes = new Uint8Array(Module.HEAPU8.buffer, _debugger_buffer, _debugger_buffer_len);    
```

Comment:
Add a comment so that someone doesn't move it by accident?

Reply:
done.